python/pyproject.toml

```diff
@@ -24,12 +24,20 @@ srt = ["aiohttp", "decord", "fastapi", "hf_transfer", "huggingface_hub", "intere
        "packaging", "pillow", "psutil", "pydantic", "python-multipart",
        "torch", "torchao", "uvicorn", "uvloop", "zmq",
        "vllm==0.5.5", "outlines>=0.0.44", "modelscope"]
+#xpu is not enabled in public vllm and torch whl,
+#need to follow https://docs.vllm.ai/en/latest/getting_started/xpu-installation.htmlinstall vllm
+srt_xpu = ["aiohttp", "decord", "fastapi", "hf_transfer", "huggingface_hub", "interegular",
```

Comment:
do not duplicate it. extract the common part out.

Reply:
Done